### PR TITLE
Add portion

### PR DESCRIPTION
**Pull request by egemenimre**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/portion/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    - portion
+
+about:
+  home: "https://github.com/AlexandreDecan/portion"
+  license: "LGPL-3.0-only"
+  license_family: LGPL
+  license_file: LICENSE.txt
+  summary: "Python data structure and operations for intervals"
+  doc_url: 
+  dev_url:
```
Comment (marcelotrevisani) [inline recipes/portion/meta.yaml:34]:
```suggestion
```

File: recipes/portion/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+about:
+  home: "https://github.com/AlexandreDecan/portion"
+  license: "LGPL-3.0-only"
+  license_family: LGPL
+  license_file: LICENSE.txt
+  summary: "Python data structure and operations for intervals"
+
+extra:
+  recipe-maintainers:
+    - AlexandreDecan
```
Comment (marcelotrevisani) [inline recipes/portion/meta.yaml:37]: @AlexandreDeca
Could you please confirm that you also want to maintain this recipe?

Comment (egemenimre) [reply recipes/portion/meta.yaml:37]: I think the other maintainer should be: @AlexandreDecan (typo in the name)

Comment (marcelotrevisani) [reply recipes/portion/meta.yaml:37]: Sorry! exactly that :smile:

Comment (AlexandreDecan) [reply recipes/portion/meta.yaml:37]: I confirm ;-)

**Pull request merged**